resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -60,13 +65,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(jsonLogin);
   }
 
-  /* checks if email_key exists in the database of queryType */
-  public boolean validEmail(String queryType, String email_key) {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userTypeString = ServletHelpers.getParameter(request, "user-type", "");
+    this.userType = userTypeString.equals("Reviewer") ? UserType.REVIEWER : UserType.REVIEWEE;
+    response.sendRedirect("/index.html");
+  }
+
+  /* checks if email_key exists in the database of userType */
+  public boolean validEmail(UserType userType, String email_key) {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Query query = new Query(queryType);
+    Query query;
+    if (userType.toString().equals("REVIEWEE")) {
```

Comment:
You don't need to convert the enum to a string.

`if (userType == UserType.REVIEWEE)`